logstash.js

```diff
@@ -23,7 +24,12 @@ module.exports = format(info => {
     delete info.timestamp;
   }
 
-  logstash['@fields'] = info;
+  if (info.level) {
+    logstash.level = info.level;
+    delete info.level;
```

Comment:
@tomasAlabes Thanks and sorry for slow review!  Only question is: why are you deleting info.level?  This doesn't seem necessary and feels like it could interfere with subsequent formatters that expect to find it?  Let me know what you think -- thanks!

Reply:
No problem David. 
Honestly, I was just following the pattern of the previous code 🤷‍♂️